basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,271 @@
 const funcs = require('./project-2');
 
 // whoops.. there is no test suite for this file. You'll simply just have to create one :/
+
+describe('project-2.js', () => {
+    
+    //TEST FOR FUNCTION ** getBiggest **
+    describe('getBiggest', () => {
+        it('should return biggest number out of two numbers', () => {
+            const expected = 20;
+            const actual = funcs.getBiggest(10, 20);
+            expect(actual).toBe(expected);
+        });
+        
+        it('should return null when any of arguments are not number', () => {
+            expect(funcs.getBiggest({}, 'ty')).toBe(null)
+            expect(funcs.getBiggest('s', [])).toBe(null)
+            expect(funcs.getBiggest('s', 'ty')).toBe(null)
+        })
+
+        it('should return undefined when any of arguments is not entered..', () => {
+            expect(funcs.getBiggest()).toBe(undefined);
+            expect(funcs.getBiggest(20)).toBe(undefined)
+        })
+    });
+
+    //TEST FOR FUNCTION ** greeting **
+    describe('greeting', () => {
+        it('should greet according to language entered..', () => {
+            const expected = 'Guten Tag!';
+            const actual = funcs.greeting('German');
+            expect(actual).toBe(expected);
+        });
+
+        it('should return only Hello! if entered other language than GERMAN and SPANISH', () => {
```

Comment:
You can test each language output to ensure everything output correctly, it's tedious, but it's thorough.